extensions/amp-video/amp-video.md

```diff
@@ -25,7 +25,11 @@ limitations under the License.
     <td class="col-fourty"><strong>Availability</strong></td>
     <td>Stable</td>
   </tr>
-   <tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
```

Comment:
Let's add this after the the validator changes are pushed to production. Otherwise including the script would cause validation errors.